src/Components/MainPage/assets/MainPageStyledComponents.ts

```diff
@@ -0,0 +1,90 @@
+import styled from 'styled-components';
+import { CardWrapper } from 'src/Components/styles/styles';
+
+export const LogoutForm = styled(CardWrapper)`
+  display: inline-block;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  margin-bottom: 1rem;
+  margin-right: 1rem;
+  position: absolute;
+  right: 0;
+`;
+export const GenreCard = styled.div<{ isChecked?: boolean }>`
+  width: 25%;
+  background: ${(props) => (props.isChecked ? '#afd2ff' : '#e7eff9')};
+  box-shadow: ${(props) => (props.isChecked ? '0px 0px 10px gray' : '0px')};
+  align-items: center;
+  border: 1px solid gray;
+  border-radius: 5px;
+  margin: 5px;
+  padding: 5px;
+  cursor: pointer;
+`;
+export const GenresWrapper = styled.div`
+  display: flex;
+  flex-wrap: wrap;
+  width: auto;
+  justify-content: center;
+`;
+
+export const MovieWrapperBlock = styled.div`
+  display: grid;
+  margin-left: 2rem;
+  margin-right: 2rem;
+  grid-template-columns: repeat(4, 1fr);
+`;
+
+export const MovieWrapperList = styled.div<{ isBlockView: boolean }>`
+  display: flex;
+  flex-wrap: wrap;
+
+  ${(props) =>
+    props.isBlockView ? 'flex-direction: row' : 'flex-direction: column'};
```

Comment:
```suggestion
  flex-direction: ${(props) =>
    props.isBlockView ? 'row' : 'column'};
```

так короче и нет дуюлирующихся слов